### ISSUE TITLE
exim_exporter depends on systemd; compiling on other *nix impossible

### ISSUE THREAD

**Issue by ltning** [gvengel/exim_exporter#4]
I would like to build/use exim_exporter on FreeBSD. However, depending on systemd makes it impossible to use exim_exporter on any other platforms than systemd-equipped Linux variants.

**Comment by gvengel** [comment]
Ah, good point! I'll try to find some time to get setup on BSD and get everything working.

**Comment by ltning** [comment]
That would be brilliant!
If you could start by removing the compile-time dependency on systemd and add an alternative way to "tail" the exim log file(s) that can be specified on startup, that would probably go a long way without requiring you to set up a BSD environment.

But hey, I won't get in the way if you want to :)

**Comment by gvengel** [comment]
Okay, I've pushed version 1.2.1 which makes systemd optional and disabled by default. It builds cleanly on FreeBSD 12, and passes all the unit tests. I also fired up a real exim server, and didn't run into any issues. Though, you will probably need point the exporter at FreeBSD's log paths.

```
./exim_exporter --exim.mainlog /var/log/exim/mainlog --exim.rejectlog /var/log/exim/rejectlog --exim.paniclog /var/log/exim/paniclog
```

Guess I should probably add an option for `--exim.logdir` or something. 😄
